src/README.md

```diff
@@ -0,0 +1,9 @@
+# SRC
+
+The shared code in this repository is composed of:
+
+- `assets` - static JavaScript initialising cookie settings and Google Analytics (UA4)
+  - note: deprecated by [@govuk-one-login/frontend-analytics](https://www.npmjs.com/package/@govuk-one-login/frontend-analytics)
+- `components` - Nunjucks templates page layout and general components
+- `lib` - Express middleware functions, designed to work using the `req`/`res`/`next` pattern of Express functions
+- `routes` - standard OAuth routes, implementing a standard flow into and out of a CRI
```

Comment:
[nit] I see you've added READMEs for lib and routes what do you think about linking them here?

Reply:
Done.